api/config/config_prod.json

```diff
@@ -152,9 +152,12 @@
   },
   "reporting": {
     "dataset": "reporting_prod",
-    "maxRowsPerInsert": 500,
+    "maxRowsPerInsert": 10000,
     "batchSizeOverrides": {
-      "dataset_domain_value": 50000
+      "dataset_domain_value": 50000,
```

Comment:
This table is "tall and narrow" - many rows, few columns.  This means that a small batch size is too slow.